ui/readme.en.md

```diff
@@ -162,11 +158,41 @@ The third type of query detection involves identifying queries with a high likel
 ![neutral relevance](https://raw.githubusercontent.com/searchhub/searchhub.github.io/master/ui/img/neutral_relevance.png)  
 The fourth type of query detection involves identifying queries with a high likelihood of medium relevance issues. In these cases we did not measure any significant user interaction with the results.
 
+## Underperforming queries
+
+The chart "Trend over Analysis Period" shows the proportion of search queries (unique per session) with particularly low KPIs across all search queries. For CTR, CR, and Findability™, this represents the lowest quantile, while for "No Results," it includes all queries with a no-hit rate of over 50% on the respective day.
+
+The table "Search Terms with No Results" lists search terms that have a no-hit rate of over 50%. Two views are available:
+
+1. Overall
+   The default setting Overall includes all search terms that resulted in no hits in at least 50% of the cases.
+2. Recent
+   The Recent setting further narrows down the list. It only shows search terms that, on the last day they were recorded, resulted in no hits in at least 50% of the cases. If a no-hit search term has been newly added to a cluster with a MasterQuery during the analysis period, it will still be displayed here for that period.
+
+The tables "Search Terms with Low CTR," "Search Terms with Low Findability™," and "Search Terms with Low CR" show the search terms in the weakest quantile of the respective KPI. The dynamically determined upper limit of the quantile for the analysis period is indicated as a percentage. This also means that the displayed percentage values apply only to the selected analysis period.
+
+The table "Search Terms with Low CR" is additionally filtered to include search terms with a CR greater than 0%.
+
+## Performance Charts{#performance-charts}
```

Comment:
Grundsätzlich benötigt man in der `en` Readme keine `{...}` hinter einer überschrift, es sei denn man möchte den automatisch generierten Link umbenannt haben, siehe Popular Queries oben.